### PR TITLE
#8 Admin header/menu

### PR DESCRIPTION
**Pull request by ExiRain**
- Updated version to 0.28
- Fixed translation of menu
- Fixed menu collapsing.
- Updated header input parameters
- Removed storestate file no longer used
- Removed UserSettings no longer used
- Refactored Generate UUID utility class
- Removed commented code from layout

### CODE REVIEW

File: src/exportcomponents/src/header/utils/generateUEID.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const generateUEID = () => {
+  let first: string | number = (Math.random() * 46656) | 0;
+  let second: string | number = (Math.random() * 46656) | 0;
```
Comment (1AhmedYasser) [inline src/exportcomponents/src/header/utils/generateUEID.ts:3]: Can you enhance this to be a one liner instead of having 2 variables for more efficiency

**Review by 1AhmedYasser — CHANGES_REQUESTED** [review]

File: src/components/layout/layout.tsx [hunk 2]
```diff
@@ -1,17 +1,23 @@
 import { FC, useState } from 'react';
 import { Outlet } from 'react-router-dom';
 // import MainNavigation from 'components/main-navigation/main-navigation.tsx'
+import items from '../main-navigation/menu.json'
+// import items from '../../exportcomponents/src/menu/components/main-navigation/menu.json'
```
[the remark this answers is not in the document]
Comment (ExiRain) [reply src/components/layout/layout.tsx]: Comments removed

File: src/exportcomponents/src/header/types/storeState.ts [hunk 3]
```diff
@@ -1,4 +1,6 @@
-export interface UserInfo {
+import { UserInfo } from "./userInfo"
+
+/*export interface UserInfo {
```
[the remark this answers is not in the document]
Comment (ExiRain) [reply src/exportcomponents/src/header/types/storeState.ts]: This file removed with one it was used in

File: src/exportcomponents/src/header/utils/generateUEID.ts (hunk 1 above)
Comment (ExiRain) [reply src/exportcomponents/src/header/utils/generateUEID.ts]: Refactored this helper

**Review by 1AhmedYasser — APPROVED** [review]

**Pull request merged**